From d8df91cce897e3c55c981e580922f5bdfc457513 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Fri, 11 Feb 2011 12:36:58 +0200
Subject: [PATCH] Documentation/Makefile: mandir usr/share xmlto fix
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Documentation/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index 9fc1165..7c84edf 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -12,7 +12,7 @@
 prefix?=$(PREFIX)
 bindir?=$(prefix)/bin
 htmldir?=$(prefix)/html
-mandir?=$(prefix)/man
+mandir?=$(prefix)/share/man
 man1dir=$(mandir)/man1
 man7dir=$(mandir)/man7
 # DESTDIR=
@@ -71,7 +71,7 @@
 	$(ASCIIDOC) -b xhtml11 -d manpage -f asciidoc.conf $(ASCIIDOC_EXTRA) $<
 
 %.1 %.7 : %.xml
-	xmlto -m callouts.xsl man $<
+	xmlto --skip-validation -m callouts.xsl man $<
 
 %.xml : %.txt footer.txt version.txt
 	$(ASCIIDOC) -b docbook -a version=$(VERSION) -d manpage -f asciidoc.conf $<
-- 
1.7.4

