From 0ff1b25d3229b3719e1afe68cadfd1db45d4bab2 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 29 Jan 2012 17:49:47 +0200
Subject: [PATCH] Makefile: run install-doc
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index b38c1e4..66df509 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@
 	@echo "Try make install"
 
 .PHONY: install
-install:
+install: install-doc
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin/
 	$(INSTALL) -m 755 guilt $(DESTDIR)$(PREFIX)/bin/
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/lib/guilt/
-- 
1.7.5.1

