From 450f18321b12f28aa5c45e78ce7b2a9bff6d984f Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Thu, 30 Aug 2007 23:27:06 +0300
Subject: [PATCH] Documentation/Makefile: add missing ASCIIDOC_EXTRA variable

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Documentation/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index 271b174..44780a8 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -73,4 +73,4 @@ usage-%.txt: $(MAN1_TXT) usage.sh
 	xmlto -m callouts.xsl man $<
 
 %.xml : %.txt footer.txt version.txt
-	$(ASCIIDOC) -b docbook -d manpage -f asciidoc.conf $<
+	$(ASCIIDOC) -b docbook -d manpage -f asciidoc.conf $(ASCIIDOC_EXTRA) $<
-- 
1.5.3.rc5

